src/components/m-header-navigation/js/stroke.js

```diff
@@ -25,7 +25,7 @@ class Stroke extends UiEvents {
       toggleClass: Stroke.DEFAULTS.toggleClass,
       closeClass: 'js-header-navigation-close',
       sameClickClose: !options.simpleMenu,
-      useDefaultEvent: !!options.simpleMenu,
+      useDefaultEvent: true,
```

Comment:
We need to make sure that this works with axa.ch, because it's package.json just requires the `beta` version and thus will always update on next deployment.

Reply:
yes agree. if we merge and deploy today i will check tomorrow. It works so far on the o-header and the SG header